data/campaigns/Winds_of_Fate/units/Crocodile.cfg

```diff
@@ -0,0 +1,136 @@
+#textdomain wesnoth-wof
+
+#define CROC_TERRAIN_FILTER_TEMP
+    terrain_type=!,*^B*,Cme*^*,Kme*^*,Wwr*^*,Wwf*^*,!,Chs*^*,Chw*^*,Cm*^*,Km*^*,W*^*,S*^*,*^Vm
+#enddef
+
+[unit_type]
+    id=Crocodile
```

Comment:
is this unit really necessary? The unit file looks identical to the core's "Swamp Lizard" unit with the following differences:
1. id and name are "Crocodile" and not "Swamp Lizard"
2. Cost is 36
3. MP is 6, and not 7.

Reply:
The crocodile also has 60 HP (rather than 50) and a different description.

The swamp lizard has a weird name and description, and these both almost make it sound confusingly like a saurian unit of some kind. It also may be frustratingly fast over water/swamp/sand/village for a unit with an invisibility type ability, allowing it to deliver a tomato surprise over a very wide area surrounding just a small patch of swamp.

Otherwise, there is no reason not to switch to the 1.15.x mainline version at this point.

Reply:
After some testing it seems the swamp lizard plays similar enough to the crocodile, both as an antagonist and a protagonist unit. So I fully replaced the crocodile with the swamp lizard.